main/src/library/Chain.flix

```diff
@@ -114,7 +114,8 @@ instance Collectable[Chain] {
     pub def collect(iter: Iterator[a, ef, r]): Chain[a] \ { ef, r } with Order[a] = Iterator.toChain(iter)
 }
 
-instance Iterable[Chain] {
+instance Iterable[Chain[a]] {
+    type Elm[Chain[a]] = a
```

Comment:
I think instead of writing out the function, we should just write:

```
type Elm = a
```

Does that make sense/work?

Reply:
Done all.